feat(ui): implementation of layout for locked toggle

vimercati-samir opened the pull request:
What this PR does 📖
Implementation of layout for locked toggle

Which issue(s) this PR fixes 🔨
AP-530

Special notes for reviewers 🗒️

Additional comments 🎤

josephmcg commented on components/views/settings/pages/privacy/Privacy.html:
```diff
@@ -11,7 +11,7 @@
         :text="$t('pages.privacy.register.subtitle')"
       >
-        <InteractablesSwitch :isEnabled="true" />
+        <InteractablesSwitch :isLocked="true" />
```
rather than `:isLocked="true"`, you could also just provide `<InteractablesSwitch isLocked />`

vimercati-samir replied:
modified accordingly in commit https://github.com/Satellite-im/Core-PWA/pull/1167/commits/92cd73802d99ee21bec2f91a0abfc0faabda5d60

josephmcg commented on components/interactables/Switch/Switch.less:
```diff
@@ -33,6 +33,14 @@
       }
```
we can improve the readability of this file, taking advantage of less. I'll DM you a newly formatted version

vimercati-samir replied:
modified accordingly in commit https://github.com/Satellite-im/Core-PWA/pull/1167/commits/92cd73802d99ee21bec2f91a0abfc0faabda5d60

josephmcg reviewed (changes requested):
functionality is good, minor refactor requests to make things more readable

josephmcg commented on components/interactables/Switch/Switch.vue:
```diff
@@ -34,6 +34,11 @@ export default Vue.extend({
       required: false,
       default: '',
     },
+
+    /**
+     * If provided the button would appear as disabled and non interactable
+     */
+    isLocked: Boolean,
```
also good to specify more details like required & default

vimercati-samir replied:
modified accordingly in commit https://github.com/Satellite-im/Core-PWA/pull/1167/commits/92cd73802d99ee21bec2f91a0abfc0faabda5d60

stavares843 commented:
<img width="716" alt="Captura de ecrã 2022-01-28, às 19 20 48" src="https://user-images.githubusercontent.com/29093946/151608365-4de8a0af-a6b5-418a-9443-93df16814d99.png">

this is the design from liz

vs branch

<img width="531" alt="Captura de ecrã 2022-01-28, às 19 21 26" src="https://user-images.githubusercontent.com/29093946/151608463-510e5e50-864d-4cc7-a23c-1b954e161bf5.png">

it should have a space between the border and the rest of the toggle

stavares843 commented:
cc @vimercati-samir

stavares843 commented:
/rebase

vimercati-samir commented:
changed accordingly and rebased
<img width="671" alt="Screenshot 2022-02-02 at 17 44 09" src="https://user-images.githubusercontent.com/52865716/152200267-2779799f-a293-4eac-a3e4-764008793718.png">

WanderingHogan commented:
/rebase

WanderingHogan commented:
Samir and I talked about how to rebase this and open a new pull request or update this one with a force push, so we need to hold off merging it
